src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,36 @@
+
+.content__container {
+    position: relative;
+    margin: 100px auto;
+  }
+
+  .clock {
+    @include circle($watch_size);
+
+    border: 3px solid black;
+    background-image: url(https://cdn.pngsumo.com/clock-clipart-transparent-4-clipart-portal-clock-clipart-transparent-3000_3000.png);
```

Comment:
smart 👍 ! 
I tried to insert numbers as pseudo elements. what a traumatic experience